src/runtime/messages.ts

```diff
@@ -0,0 +1,81 @@
+import * as proto from "../proto/server/v1/realtime";
+// import { encode, decode } from "@ably/msgpack-js";
```

Comment:
instead of this, should we use this https://github.com/msgpack/msgpack-javascript

Reply:
Sure. I don't really have a strong opinion about it.

Reply:
I've switched to this library. It works well.